src/popup/router/components/InputField.vue

```diff
@@ -142,6 +143,12 @@ export default {
       return !!this.message;
     },
   },
+  methods: {
+    checkIfNumber(event) {
+      if (this.$attrs.type !== 'number' || ['Enter', 'Backspace'].includes(event.key)) return;
```

Comment:
Please avoid using `$attrs` in the component logic as it makes the component less predictable. If we want this component to be able to receive some data it has to have the prop for this.

Also please use consistent returns within the scope of the functions. I know that early returns are tempting, but it won't work in typescript, which will be used in some time.

I think merging these two if statements can would be the best solution.